rest-api/flask_api/app.py

```diff
@@ -142,5 +142,25 @@ def normalize_datatypes(data):
     dc['hospitalization_rate'] = to_float(dc, 'hospitalization_rate')
     dc['latitude'] = to_float(dc, 'latitude')
     dc['longitude'] = to_float(dc, 'longitude')
-    
+
     return dc
+
+
+@app.route('/prediction/<int:location_id>')
+def fetch_prediction_data(location_id):
+    # placeholder query since my understanding is we need to update db columns/tables
+    # to include the regions described in the predictive model
+    sql = """
+        SELECT DISTINCT f.date_id, f.location_id, cases, recoveries, deaths,
+        cases_100k, testing_rate, hospitalization_rate,
+        date, year, month, day_of_week, day_of_month,
+        country, state, city, latitude, longitude, population
+        FROM covid_facts f JOIN date_dim d ON d.date_id = f.date_id
+        JOIN location_dim l ON l.location_id = f.location_id
+        WHERE f.location_id = %s;
+        """
```

Comment:
I think we only need the last 15 or so days of data to feed into the model so I think should tack on an 

```
ORDER BY date LIMIT 15
```